tsconfig.json

```diff
@@ -1,20 +1,20 @@
 {
-    "include": ["src", "types"],
+    "include": ["src"],
     "compilerOptions": {
         "module": "esnext",
         "lib": ["dom", "esnext"],
         "importHelpers": true,
         "declaration": true,
-        "declarationDir": "dist/types",
+        "declarationDir": "types",
         "sourceMap": true,
-        "rootDir": "./src",
         "strict": true,
         "noImplicitReturns": true,
         "noFallthroughCasesInSwitch": true,
         "noUnusedLocals": true,
         "noUnusedParameters": true,
         "moduleResolution": "node",
         "jsx": "react",
+        "removeComments": false,
```

Comment:
This is specifically the TS comments which will show in your `.d.ts`, the build process should remove any comments from the actual `dist` code.

Reply:
Thank you for this one!